src/notebook/widget.ts

```diff
@@ -201,6 +200,7 @@ class NotebookWidget extends Widget {
       widget = new BaseCellWidget(cell);
     }
     widget.addClass(NB_CELL_CLASS);
+    widget.input.editor.addClass(NB_EDITOR_CLASS);
```

Comment:
Or really, just that if the editor widget is swapped out, you should copy the classes over.